apps/web-giddh/src/app/add-company/add-company.component.ts

```diff
@@ -1247,6 +1252,9 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
             delete this.company.permission;
             delete this.company.creatorSuperAdmin;
         }
+        if (this.secondStepForm.value.businessNature === "Other") {
+            this.company.otherBusinessNature = this.secondStepForm.value.otherBusinessNature;
```

Comment:
Either use controls to get value or this way. don't use both ways. follow 1 pattern only

Reply:
Done